src/chrome/content/rules/QQ.xml

```diff
@@ -281,7 +278,12 @@
 	<target host="i.mail.qq.com" />
 	<target host="res.mail.qq.com" />
 	<target host="rl.mail.qq.com" />
+	<target host="map.qq.com" />
+		<exclusion pattern="^http://map\.qq\.com/(?!api/)" />
+		<test url="http://map.qq.com/api/js" />
```

Comment:
I couldn't reproduce the rewriting, but I just realized that it is not a problem for http site importing a https script which loads another http script...

BTW, it is excluded in 93f4a12